collect_app/src/main/java/org/odk/collect/android/widgets/SelectMultiWidget.java

```diff
@@ -132,6 +140,15 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                         buttonView.setChecked(true);
                     }
                 }
+
+                // show warning when selected choice value has spaces
+                int index = (int) checkBox.getTag();
+                String value = items.get(index).getValue();
+                if (isChecked && value != null && value.contains(" ")) {
+
+                    String warning = context.getString(R.string.invalid_space_in_answer_singular, value);
+                    ToastUtils.showLongToast(warning);
```

Comment:
I suggest you inline `warning`. I don’t see the value in having a variable for it.